internal/cmd/dev/dev.go

```diff
@@ -94,20 +92,9 @@ func runDevScript(ctx context.Context, app meta.App) error {
 func startDevServer(ctx context.Context, args []string) error {
 	app := meta.New(".")
 
-	cfgFile := "./.buffalo.dev.yml"
```

Comment:
`dev` command is not an anywhere command, so if the command was executed from the directory that has no `.buffalo.dev.yml`, the execution will be aborted by the root's pre-checking and this block should not be reachable.